freud/plot.py

```diff
@@ -142,20 +196,75 @@ def pmft_plot(pmft, ax=None):
         ax.yaxis.set_ticks([i for i in range(int(ylims[0]), int(ylims[1]+1))])
         ax.set_xlabel(r'$x$')
         ax.set_ylabel(r'$y$')
-
         ax.set_title('PMFT')
+
+        ax_divider = make_axes_locatable(ax)
+        cax = ax_divider.append_axes("right", size="7%", pad="10%")
```

Comment:
This is a cool feature, I've struggled with sizing colorbars onto existing figures before.

Reply:
This is pretty weird. So there are two almost same classes, `mpl_toolkits.axes_grid1.colorbar.colorbar` and `mpl_toolkits.axes_grid1.colorbar.Colorbar`. 
The documentation says `colorbar` is a `pyplot` thin wrapper while `Colorbar` is not.
So I believe using `colorbar` as in this [example](https://matplotlib.org/3.1.0/gallery/axes_grid1/demo_colorbar_with_axes_divider.html) causes memory leak since it calls `matplotlib.pyplot`.

Reply:
I believe this is also the reason why the `_repr_png_()` for `PMFT2DXY` gave another empty plot, since the empty plot is probably from `matplotlib.pyplot`.

Reply:
This is the bug that existed, of having an empty figure below PMFT plot.
<img width="464" alt="Screen Shot 2019-06-19 at 10 27 54 PM" src="https://user-images.githubusercontent.com/15791520/59813961-89e08380-92e1-11e9-8de4-4becc25d97b8.png">

And I think the below example shows that the bug was caused from `matplotlib.pyplot` called from `colorbar` since the `plt.plot()` is actually shown below the PMFT plot, where there used to be an empty figure.
<img width="478" alt="Screen Shot 2019-06-19 at 10 28 01 PM" src="https://user-images.githubusercontent.com/15791520/59813966-8d740a80-92e1-11e9-877b-afc615a707fc.png">

Reply:
@jinsoo960 Good catch. If this is fixed, please resolve this conversation.